src/main/java/com/gtnewhorizons/angelica/transform/RedirectorTransformer.java

```diff
@@ -44,6 +44,8 @@ public class RedirectorTransformer implements IClassTransformer {
     private static final String GL11 = "org/lwjgl/opengl/GL11";
     private static final String GL13 = "org/lwjgl/opengl/GL13";
     private static final String GL14 = "org/lwjgl/opengl/GL14";
+    private static final String PROJECT = "org/lwjgl/util/glu/Project";
```

Comment:
Don't you need to add this in the class constant pool parser ?

Reply:
Probably should, however the place it was being used was already identified by one of the other classes.